modules/next/src/Controller/NextSiteEntityController.php

```diff
@@ -51,30 +51,45 @@ public function environmentVariables(NextSiteInterface $next_site) {
     $build = [];
 
     $variables = [
+      'required_variables' => '# Required',
       'NEXT_PUBLIC_DRUPAL_BASE_URL' => $this->request->getSchemeAndHttpHost(),
       'NEXT_IMAGE_DOMAIN' => $this->request->getHost(),
-      'DRUPAL_SITE_ID' => $next_site->id(),
-      'DRUPAL_FRONT_PAGE' => $this->config('system.site')->get('page.front'),
     ];
 
     if ($secret = $next_site->getPreviewSecret()) {
       $variables += [
+        'preview_variables' => '# Required for Preview Mode',
         'DRUPAL_PREVIEW_SECRET' => $secret,
-        'DRUPAL_CLIENT_ID' => 'Retrieve this from /admin/config/services/consumer',
-        'DRUPAL_CLIENT_SECRET' => 'Retrieve this from /admin/config/services/consumer',
       ];
     }
 
+    $variables += [
+      'authentication_bearer' => '# Authentication (Bearer)',
+      'DRUPAL_CLIENT_ID' => 'Retrieve this from /admin/config/services/consumer',
+      'DRUPAL_CLIENT_SECRET' => 'Retrieve this from /admin/config/services/consumer',
+    ];
+
+    $variables += [
+      'optional_variables' => '# Optional',
```

Comment:
If these are included in the environment variables, should these be documented in https://next-drupal.org/docs/environment-variables?

Reply:
Yeah I think we can document these as optional or deprecated since they are not used anymore. Could be helpful as reference.

Reply:
@lauriii https://next-drupal-git-feat-env-variables-chapter-three.vercel.app/docs/environment-variables

Reply:
Looks good 👍